static/app/actionCreators/pageFilters.tsx

```diff
@@ -198,6 +215,9 @@ export function initializeUrlState({
   // Use period from default if we don't have a period set
   pageFilters.datetime.period ??= defaultDatetime.period;
 
+  const {...codecovDefaultFilters} = getCodecovDefaultSelection();
+  const codecovPageFilters: CodecovPageFilters = {...codecovDefaultFilters};
```

Comment:
probably don't need to clone here either since getCodecovDefaultSelection makes a new object every time
```suggestion
  const codecovPageFilters: CodecovPageFilters = codecovDefaultFilters;
```